src/intermediate_structures.cpp

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file intermediate_structures.cpp
+ * @brief Support functions for the intermediate structures
+ *
+ * @copyright Copyright (C) 2021  Andrew Ritchie
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+
+#include "intermediate_structures.h"
+#include <string>
+
+std::string PostProcessing::stringTrustworthy(Status s) {
+  switch (s) {
+    case Trustworthy:
+      return "Yes";
+    case Untrustworthy:
+      return "No";
+    default:
+      return "Unknown";
+  }
+}
```

Comment:
Could we move this into a `PostProcessing` namespace to match the `Status` enum's namespace?

Reply:
Yes, no problem! :)